### PR TITLE
More hashcat rules

### PR DESCRIPTION
**Pull request by magnumripper**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -247,7 +247,7 @@ indent_size = 8
 [vax.[ch]]
 indent_size = 8
 
-[wordlist.[ch]]
```
Comment (frank-dittrich) [inline .editorconfig]: This seems unrelated. Was it intended?

Comment (magnumripper) [reply .editorconfig]: Yes, it was "needed" for the little change in `wordlist.c`. Our version of that file is so different from the original I decided to simply use the "jumbo indentation" of just 4, rather than trying to break up lines just making them less readable.
